### ISSUE TITLE
Not possible to select 'English' interface on Polish Windows.

### ISSUE THREAD

**Issue by Klopot** [LiMinggang/madedit-mod#110]
Same was on orginal MadEdit:there is no option to choose English interface on Polish Windows machine.
Using 'system default' default gives Polish (as expected).
Only way out is to delete 'locale' folder content.

![no_eng](https://cloud.githubusercontent.com/assets/16302984/11806987/f239dcba-a316-11e5-962c-0ac20670c1e6.png)

**Comment by LiMinggang** [comment]
Yes. It's a bug. It has been fixed it by "English was missing in the language list #105".

If it was a blocking issue for you, I can provide a nightly build for you. What's OS are you using?

**Comment by Klopot** [comment]
Sorry, i've missed #105. 
It's not a big issue since deleting 'locale\*.*' fixes interface to English - don't bother doing nightly build just for that.
I have just luck in bug finding for everything I touch ;-)
MadEdit seems to be real great piece of software - You're doing great job!
Tested on Win 7 x64 PL Pro [6.1.7601]
Thanks and regards
PK
Dnia 15 grudnia 2015 13:48 Minggang Li &lt;notifications@github.com&gt; napisał(a):
Yes. It's a bug. It has been fixed it by "English was missing in the language list #105".If it was a blocking issue for you, I can provide a nightly build for you. What's OS are you using?—

**Comment by LiMinggang** [comment]
Thanks for your support. You must be a excellent geek.
BTW: I am working on Win 7 64bit. So, it's not a big duel to provide a build since I would build several times on it for bug fix or new idea.

The only issue is that the build might has new bugs.

**Comment by Klopot** [comment]
I prefer: service engineer ;-)
(that's my job too)
Since ME has many features I like, I can sometimes be alfa/beta tester - no problem.
I'll try to report bugs if I find any.
Regards
PK
Dnia 15 grudnia 2015 14:24 Minggang Li &lt;notifications@github.com&gt; napisał(a):
Thanks for your support. You must be a excellent geek.
BTW: I am working on Win 7 64bit. So, it's not a big duel to provide a build since I would build several times on it for bug fix or new idea.The only issue is that the build might has new bugs.—

**Comment by LiMinggang** [comment]
Thanks.
I'm doing performance improvement recently after I found someone did a test
on MadEdit-Mod at
http://www.portablefreeware.com/forums/viewtopic.php?f=2&t=22061&start=75.
Currently, I think(just by observation, no exact data) current code almost
doubled the performance for loading or reformatting by switching back and
force from/to text mode for text file.

Will release it in 0.4.0 or 0.3.11 if I had no idea for the 0.4.x.

If you often had something done in the same way, don't hesitate to try the
scripting/macro functions like what I did. It saved me a lot!
